src/pages/OrchestrationPage/index.tsx

```diff
@@ -0,0 +1,161 @@
+            if (valueItem.type == 'checkbox') {
+              valueItem.isChecked = !valueItem.isChecked
+            }
+          })
+        }
+      })
+    }
+    const tableDataUpdated: { headers?: column[]; rows?: row[] } = {
+      headers: tableData && tableData?.headers,
+      rows: tableData && tableData?.rows,
+    }
+    setTableData({ ...tableDataUpdated })
+  }
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:8080/table_data')
+      .then((res) => {
+        console.log('Result is : ', res)
+
+        setTableData({
+          headers: res.data.headers,
+          rows: res.data.data,
+          header_names: res.data.header_names,
+          result: res.data.result,
+        })
+        setRowsLength(res.data.result.length)
+      })
+      .catch((err) => console.log(err))
+  }, [])
+
+  return (
+    <Box sx={{marginBottom : "10px"}}>
+      <Box
+        sx={{
+          borderBottom: '1px solid #eff0f3',
+          marginBottom: '40px',
+          pb: '6px',
+        }}
+      >
+        <Typography>
+          Account<span style={{ marginLeft: '40px' }}>:</span> Coffee Company
+        </Typography>
+        <Typography>
+          Account ID<span style={{ marginLeft: '20px' }}>:</span> 97034
+        </Typography>
+      </Box>
+      <Box sx={{ display: 'flex', alignItem: 'center', marginBottom : "40px" }}>
+        <CustomizedInputBase
+          placeholder={'Search'}
+          inputValue={'Search'}
+          onChange={() => {}}
+        />
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'flex-end',
+            alignItems: 'center',
+            flex: 1,
+          }}
+        >
+          <CustomButton
+            variant="outlined"
+            children={
+              <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                <Typography
+                  sx={{ color: 'black', fontWeight: 'bold', pr: '5px' }}
+                  variant="body2"
+                >
+                  Move to
+                </Typography>
+                <ArrowDownward fontSize="small" sx={{ color: 'black' }} />
+              </Box>
+            }
+            sx={{ ...moveToTemplateStyle }}
+          />
+
+          <CustomButton
+            variant="outlined"
+            children="JOURNEY TEMPLATE"
+            sx={{ ...journeyTemplateStyle, marginLeft: '40px' }}
+          />
+          <CustomButton
+            variant="contained"
+            children="CREATE NEW JOURNEY"
+            sx={{ ...createJourneyStyle, marginLeft: '40px' }}
+          />
+        </Box>
+      </Box>
+      <Table
+        headers={tableData?.headers}
+        header_names={tableData.header_names}
+        result={tableData.result}
+        rows={tableData?.rows}
+        handleCheckBox={handleCheckBox}
+        rowsLength={rowsLength}
+      />
+    </Box>
+  )
+}
+
+const OrchestrationPage = () => {
+  return <MainTemplate child={<MainContent />}></MainTemplate>
+}
```

Comment:
pages will render a template which is correct. But MainContent can be an organism. 
Pages will not have any logic